src/Swarm/Game/Scenario.hs

```diff
@@ -42,6 +42,8 @@ module Swarm.Game.Scenario (
   ScenarioCollection (..),
   scenarioCollectionToList,
   ScenarioItem (..),
+  _SISingle,
+  _SICollection,
```

Comment:
`_SICollection` does not seems to be used. Also isn't it odd to export symbol prefixed with `_`, I thought this was for reserved for internal name.